.github/workflows/ci.yml

```diff
@@ -13,10 +13,10 @@ env:
   INFURA_API_KEY: '${{ secrets.INFURA_API_KEY }}'
   USER_PRIVATE_KEY: '${{ secrets.USER_PRIVATE_KEY }}'
   ETHERSCAN_API_KEY: '${{ secrets.ETHERSCAN_API_KEY }}'
-  MAINNET_RPC: https://rpc.mevblocker.io
+  MAINNET_RPC: '${{ secrets.MAINNET_RPC }}'
```

Comment:
I have configured the GH actions secrets of the repo to point this variable to our local Erigon mainnet node.